src/gsi/cplr_get_fv3_regional_ensperts.f90

```diff
@@ -186,8 +210,143 @@ subroutine get_fv3_regional_ensperts_run(this,en_perts,nelen,ps_bar)
                         end do
                      end do
   
-  
-               end select
+! save additional ensemble varaible data for direct reflectivity DA
+
+                  case('ql','QL')
+
+                     do k=1,grd_ens%nsig
+                        do i=1,grd_ens%lon2
+                           do j=1,grd_ens%lat2
+                              w3(j,i,k) = ql(j,i,k)
+                              x3(j,i,k)=x3(j,i,k)+ql(j,i,k)
+                           end do
+                        end do
+                     end do
+
+                  case('qi','QI')
+
+                     do k=1,grd_ens%nsig
+                        do i=1,grd_ens%lon2
+                           do j=1,grd_ens%lat2
+                              w3(j,i,k) = qi(j,i,k)
+                              x3(j,i,k)=x3(j,i,k)+qi(j,i,k)
+                           end do
+                        end do
+                     end do
+
+                  case('qr','QR')
+
+                     do k=1,grd_ens%nsig
+                        do i=1,grd_ens%lon2
+                           do j=1,grd_ens%lat2
+                              if (l_use_cvpqx) then
+                                  if (qr(j,i,k) <= 1.0E-5_r_kind) then !Originally Gang used 5.0E-5
+                                      qr(j,i,k) = 1.0E-5_r_kind         !Rong Kong
+                                  end if
+                                  if (l_use_cvpqx_pval .gt. 0.0_r_kind ) then !CVpq
+                                     if(mype==0 .and. i==10 .and. j==10 .and. k==10) write(6,*)'power transform for qr : from member-->',n
```

Comment:
There are lots of write command like this. I suggest to comment out this kind of write command and 
1) change it to a comment 
2) move this out of loop k,i,j because it does not related to i,j,k.

Reply:
I modified the code according to your comments. I took out these codes from loops and make them print one time at the start of the routine using if(mype==0). Unnecessary usage of i, j, k is removed too. You may see the changes at 'gsi_rfv3io_mod.f90’, ‘cplr_get_fv3_regional_ensperts.f90’, ‘cplr_get_wrf_mass_ensperts.f90’, ‘cplr_wrwrfmassa.f90’ and so on. If you still recommend us to comment out the part to print info message, we will do that.

https://github.com/NOAA-EMC/GSI/pull/142/files#diff-86aea35b64364db7913dd1eb935d98f572e0601c925defcb5595748644029235R90-R108